src/options/quantifiers_options.toml

```diff
@@ -1995,3 +1995,39 @@ header = "options/quantifiers_options.h"
   type       = "bool"
   default    = "false"
   help       = "Enable SyGuS instantiation quantifiers module"
+
+[[option]]
+  name       = "sygusInstScope"
+  category   = "regular"
+  long       = "sygus-inst-scope=MODE"
+  type       = "SygusInstScope"
+  default    = "INSIDE"
+  help       = "select scope of ground terms"
+  help_mode  = "scope for collecting ground terms for the grammar."
+[[option.mode.INSIDE]]
+  name = "inside"
+  help = "use ground terms inside quantifier bodies only."
+[[option.mode.OUTSIDE]]
+  name = "outside"
+  help = "use ground terms outside of quantifier bodies only."
+[[option.mode.BOTH]]
+  name = "both"
+  help = "combines inside and outside."
+
+[[option]]
+  name       = "sygusInstGranularity"
```

Comment:
This option name is a bit too general sounding to me, I think `sygusInstTermSel` may be better (note this has some similarities with this option: https://github.com/CVC4/CVC4/blob/master/src/options/quantifiers_options.toml#L368).

Reply:
done